src/TogglService.php

```diff
@@ -17,7 +17,7 @@
 
 class TogglService {
 
-    use ReportTrait, ReportUtilityTrait, ClientTrait, TaskTrait, TagTrait, GroupTrait, ProjectTrait, TimeEntryTrait, WorkspaceTrait, WorkspaceUsersTrait;
+    use ReportTrait, DetailedReportUtilityTrait, ReportUtilityTrait, ClientTrait, TaskTrait, TagTrait, GroupTrait, ProjectTrait, TimeEntryTrait, WorkspaceTrait, WorkspaceUsersTrait;

```

Comment:
You forgot the import statement for this one. Fixed it myself, not a big thing. Just thought I'd let you know